quinn-proto/src/frame.rs

```diff
@@ -40,7 +40,7 @@ impl coding::Codec for Type {
     }
 }
 
-pub trait FrameStruct {
+pub(crate) trait FrameStruct {
```

Comment:
I'm not sure I agree with explicitly crate-qualifying all internal item definitions, and in any case, such a change would belong in a separate PR so it can be more easily reviewed.